### PR TITLE
Add Tokenization Collections API

### PR DESCRIPTION
**Pull request by or109**
## Pull Request Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes (link to the issue here)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore / Documentation
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Locally tested against Fireblocks API

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added corresponding labels to the PR

### CODE REVIEW

File: src/fireblocks-sdk.ts [hunk 1]
```diff
@@ -2081,6 +2088,78 @@ export class FireblocksSDK {
...
+        return await this.apiClient.issueGetRequest(`/v1/tokenization/collections/${collectionId}/tokens/${tokenId}`);
+    }
+
+    /**
+     * Burn collection token
+     *
+     * @returns WriteCallFunctionResponseDto
+     */
+    public async burnCollectionToken(collectionId: string, payload: BurnCollectionTokenRequest): Promise<WriteCallFunctionResponseDto> {
+        return await this.apiClient.issuePostRequest(`/v1/tokenization/collections/${collectionId}/tokens`, payload);
```
Comment (phanigovindaraju) [inline src/fireblocks-sdk.ts:2160]: Isnt this delete request?

Comment (phanigovindaraju) [reply src/fireblocks-sdk.ts:2160]: Ignore the comment. Just read your comment indicating this will be removed!

Comment (or109) [reply src/fireblocks-sdk.ts]: Burn removed.

File: src/types.ts [hunk 2]
```diff
@@ -1874,6 +1874,26 @@ export interface TokenLink {
...
 
+export enum CollectionType {
+    NON_FUNGIBLE_TOKEN = "NON_FUNGIBLE_TOKEN",
+    SEMI_FUNGIBLE_TOKEN = "SEMI_FUNGIBLE_TOKEN",
+}
+
+export interface CollectionLink {
+    id: string;
+    type: CollectionType;
+    refId: string;
```
Comment (phanigovindaraju) [inline src/types.ts:1885]: I believe we decided not to have refId in the CollectionLInk to avoid confusion with multiple Ids.

Comment (phanigovindaraju) [reply src/types.ts:1885]: @or109  Not sure if you got a chance to check this comment.

**Review by amimagid — APPROVED** [review]

**Pull request merged**